.github/workflows/generate-cards.yml

```diff
@@ -0,0 +1,19 @@
+name: Generate Cards
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
I like that you included the dispatch trigger as well.
So people with write access can manually trigger this on demand from the Actions tab as well!

Note: I think GitHub pauses Actions for repos that had no activity for a certain time.